game/plugins/src/main/kotlin/gg/rsmod/plugins/content/skills/thieving/pickpocket/PickpocketNpc.kt

```diff
@@ -0,0 +1,462 @@
+package gg.rsmod.plugins.content.skills.thieving.pickpocket
+
+import gg.rsmod.game.model.weight.impl.WeightItem
+import gg.rsmod.game.model.weight.impl.WeightItemSet
+import gg.rsmod.plugins.api.cfg.Items
+import gg.rsmod.plugins.api.cfg.Npcs
+
+//Rarity weighting
+private const val ALWAYS = 0
+private const val COMMON = 256
+private const val UNCOMMON = 32
+private const val RARE = 8
+private const val VERY_RARE = 1
+
+/**
+ * @npcIds = an array of NPC npcIds for them to pickpocket
+ * @experience = the amount of experienceerience given per pickpocket
+ * @lvl = the level requirement to pickpocket that npc
+ * @rewards = a weighted set of possible item rewards
+ * @damage = damage range when getting stunned
+ * @stunTicks = the amount of time that the npc stuns the player for
+ */
+enum class PickpocketNpcs(val npcIds: IntArray, val experience: Double, val lvl: Int, val npcName: String? = null,
```

Comment:
The naming should be singular (`PickpocketNpc` vs `PickpocketNpcs`) - see https://stackoverflow.com/a/15756009/5065094

Reply:
While I'm here:
- rename `npcName` just to `name`
- similarly, rename `npcIds` to `ids`
- rename `lvl` to `levelRequirement` (or similar, so that it's clear it's a thieving level requirement)

Reply:
As stated here, @Tomm0017 wanted npcIds to be like this, as it was previously ids. https://github.com/Tomm0017/rsmod/pull/59#discussion_r269592540